src/clr-addons/slider/slider.html

```diff
@@ -0,0 +1,56 @@
+<!--
+  ~ Copyright (c) 2018 Porsche Informatik. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="slider">
+  <h4>Value: {{_value}}</h4>
+  <div class="slider-action-field input" [hidden]="!_showsLabels">
+    <input class="value-boundary"
+           title="Minimum"
+           type="number"
+           name="Minimum"
+           [(ngModel)]="_minValue"
+           [readonly]="![enableValueFields]"
+           (change)="onMinValueChange($event.target.value)">
+  </div>
+
+  <!-- TODO:-->
+  <!-- .) orient does not seem to get bound to the sliders' base class orientation-->
+  <div class="slider-action-field">
+    <div
+      [class.slider-wrapper]="_orientation == 'vertical'"
+      [class.slider-size-sm]="_size == 'sm'"
+      [class.slider-size-md]="_size == 'md'"
+      [class.slider-size-lg]="_size == 'lg'"
+      [class.slider-size-xl]="_size == 'xl'">
+      <input #slider
+             class="slider-content"
```

Comment:
this is the slider itself, naming the class slider-content does not fit expectations. you can target such things in css with

.slider-wrapper {
    input[type=range] {
  }
}